File: ympa_fsharp_giraffe_server/src/model/GetGoodsFeedbackCommentsResponse.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.ApiResponseStatusType
open ympa_fsharp_giraffe_server.Model.GoodsFeedbackCommentListDTO

module GetGoodsFeedbackCommentsResponse =

  //#region GetGoodsFeedbackCommentsResponse


  type GetGoodsFeedbackCommentsResponse = {
    Status : ApiResponseStatusType;
    Result : GoodsFeedbackCommentListDTO;
  }
  //#endregion

File: src/main/scala/org/openapitools/models/GetGoodsFeedbackCommentsResponse.scala
package org.openapitools.models

import io.circe._
import io.finch.circe._
import io.circe.generic.semiauto._
import io.circe.java8.time._
import ympa_scala_finch_server._
import org.openapitools.models.ApiResponseStatusType
import org.openapitools.models.GoodsFeedbackCommentListDTO

/**
 * 
 * @param status 
 * @param result 
 */
case class GetGoodsFeedbackCommentsResponse(status: Option[ApiResponseStatusType],
                result: Option[GoodsFeedbackCommentListDTO]
                )

object GetGoodsFeedbackCommentsResponse {
    /**
     * Creates the codec for converting GetGoodsFeedbackCommentsResponse from and to JSON.
     */
    implicit val decoder: Decoder[GetGoodsFeedbackCommentsResponse] = deriveDecoder
    implicit val encoder: ObjectEncoder[GetGoodsFeedbackCommentsResponse] = deriveEncoder
}